src/PDFA-3-0/Cert/XIS-Server/medmij-pdfa-xis-1-5-serve-0-binary.xml

```diff
@@ -4,20 +4,20 @@
     nts:scenario="server">
     <id value="xis-1-5-serve-0-binary"/>
     <version value="stu3-2.0"/>
-    <name value="PDFA - XIS Server - Scenario 1.5 - Serve no Binary resource"/>
-    <description value="Scenario 1.5 - Serve no Binary resource of XXX-Baltus."/>
+    <name value="PDFA - XIS Server - Scenario 1.5 - Serve zero Binary resources"/>
```

Comment:
I think the description is still somewhat beside the point.

Reply:
I agree! (To be fixed in the context of MM-3378)